buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -56,7 +56,11 @@ object Versions {
     
     object Bukkit {
         const val minecraft = "1.21.4"
-        const val paperBuild = "$minecraft-R0.1-20250317.101324-208"
+
+        // swapped to use a generic SNAPSHOT instead of a specific timestamp
+        //const val paperBuild = "$minecraft-R0.1-20250317.101324-208"
+        const val paperBuild = "$minecraft-R0.1-SNAPSHOT"
+
```

Comment:
Do not swap to use snapshots. We do not allow generic snapshot versions in Terra. This is so that we don't have to download a new (unnecessary) version of a platform (which are often huge) every single time someone wants to work on Terra. If there is an issue simply update the time stamp :]